src/components/Common/Header.js

```diff
@@ -133,7 +140,14 @@ function Header() {
                 className={styles.tokenIcons}
                 id={open ? styles.show : styles.hide}
               >
-                <BiBell size="2vw" className={styles.bell} />
+                <div className={styles.alam}>
+                  <BiBell
+                    size="2vw"
+                    onClick={alamOpen}
+                    className={styles.bell}
+                  />
+                  {isAlamOpen ? <AlamContainer /> : null}
+                </div>
                 <FaUserCircle
                   size="2vw"
                   className={styles.Profile}
```

Comment:
```suggestion
                  className={styles.Profile}
```
이 부분도 마찬가지로 적용해주면 될 것 같아요